benchmarks/Cases/BenchmarkZstdIgbinary.php

```diff
@@ -57,27 +60,40 @@ public function seed(): void
      */
     protected function setSerialization($client): void
     {
-        $client->setOption(Redis::OPT_SERIALIZER, Redis::SERIALIZER_IGBINARY);
-        $client->setOption(Redis::OPT_COMPRESSION, Redis::COMPRESSION_ZSTD);
+        if (! $client->setOption($client::OPT_SERIALIZER, $client::SERIALIZER_IGBINARY)) {
```

Comment:
When `Redis` build without `igbinary` support there is no such constant `SERIALIZER_IGBINARY` and test fails with error
```
Fatal error: Uncaught Error: Undefined constant Redis::SERIALIZER_IGBINARY in /relay/benchmarks/Cases/BenchmarkZstdIgbinary.php:63
```
That's why I used `Relay::SERIALIZER_IGBINARY`

Reply:
Ah, should we maybe just do a constant check on the actual class to be explicit about it?

Reply:
done